practice-two/src/Components/Input/input.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+interface InputProps {
+  type: string 
+  id: string 
+  name: string
+  value?: string
+  placeholder? :string
+  className: string 
+  checked?: boolean
+  accept?: string 
+  handleInput(e: any): void
+}
+
+function Input(props: InputProps) {
+  const {type, id, name, value, className, checked, placeholder, accept, handleInput} = props
```

Comment:
`handleInput` event handler should specify event type. For example: `handleInputChange`